web/olga/analytics/tests/test_models.py

```diff
@@ -163,14 +158,11 @@ def test_students_per_country_render(self, mock_get_last_calendar_day):
         Verify that get_students_per_country method returns data to render correct values.
         """
         mock_get_last_calendar_day.return_value = date(2017, 6, 1), date(2017, 6, 2)
+        country_list, country_tab_list = InstallationStatistics.get_students_per_country()
 
-        datamap_format_countries_list, tabular_format_countries_list = self.create_expected_default_data()
-
-        result = InstallationStatistics.get_students_per_country()
-
-        self.assertEqual(
-            (datamap_format_countries_list, tabular_format_countries_list), result
-        )
+        self.assertEqual(tabular_format_countries_list, country_tab_list)
```

Comment:
I would start the variable name of the expected result with the `expected` word.